client/examples/react/src/App.tsx

```diff
@@ -4,8 +4,9 @@ import logo from "./logo.png";
 import {
   DekuToolkit,
   fromMemorySigner,
-} from "@marigold-dev/deku-toolkit";
+} from "deku-toolkit";
 import { InMemorySigner } from "@taquito/signer";
+import { JSONType } from "../../../lib/utils/json";
```

Comment:
Do not use the JSONType, it's a private type, it's not exposed by the client

Reply:
Done, thank you 🙏